src/ontology/oeo.omn

```diff
@@ -190,6 +190,18 @@ Class: OEO_00020012
         OEO_00000506 some OEO_00000064
     
     
+Class: OEO_000500000000000000000000
```

Comment:
This class ID is way to long, you've probably set Protégé incorrectly. Please check carefully whether you've exactly applied the settings described here: https://github.com/OpenEnergyPlatform/ontology/wiki/Numerical-Identifiers

For the class you've included I assume we can fix this manually.
```suggestion
Class: OEO_00050000
```

Reply:
oh yes, in my preferences there were 20 digits allowed. I changed it to 4 now.

And I just noticed that I made the changes in OEO.omn - Should it be the shared module instead?
If yes, can I delete this pull request and implement it again?

Reply:
Oh, I've missed that you included the class in the oeo.omn. I think it should be in the physical module like e.g. energy transformation.

@sfluegel05 : What do you suggest is the easiest way to shift the class from oeo.omn to the module file?

Reply:
The easiest way is to use a text editor and just move the class from one file to the other (and check with Protege afterwards if the ontology still works).

Reply:
When I copy the added lines from oeo.omn to the last rows of oeo-physical.omn I get an error message from Protégé, saying: org.xml.sax.SAXParseException; systemId: file:/C:/Users/vsehn/Documents/GitHub/ontology/src/ontology/oeo.omn; lineNumber: 1; columnNumber: 1; Content ist nicht zulässig in Prolog.

I don't understand, because I didn't change something in lineNumber 1... what can I do now?

Reply:
This looks like something on the format-level broke. Maybe you made a mistake while copying and forgot a closing bracket or something similar? I would try to reload the last functioning state of the ontology (in this case the last commit) and try again.

When I tried to do this myself, everything worked fine, so it's hard to tell the exact reason why it didn't work for you.

Reply:
I also added a new commit the moves industrial process to oeo-physical

Reply:
thanks! I think I copied also the last line, which you didn't...